apps/condo/domains/common/components/EmptyListView.tsx

```diff
@@ -68,26 +70,31 @@ export const EmptyListView: React.FC<IEmptyListProps> = (props) => {
             <Typography.Title level={4}>
                 {label}
             </Typography.Title>
-            <Typography.Text type='secondary'>
-                {message}
-            </Typography.Text>
-            <Row gutter={[10, 0]} align='middle' justify='center' style={{ marginTop: '24px' }}>
-                {
-                    button ? (
+            {
+                (accessCheck || isUndefined(accessCheck)) &&
+                <>
+                    <Typography.Text type='secondary'>
+                        {message}
+                    </Typography.Text>
+                    <Row gutter={[10, 0]} align='middle' justify='center' style={{ marginTop: '24px' }}>
```

Comment:
gutter and style props should be moved to constants out of the component body

Reply:
i just added a tab :) but git displays it as my code :)

Reply:
but ok i'll fix it